products/admin.py

```diff
@@ -1,3 +1,41 @@
 from django.contrib import admin
+from django.db.models.base import ModelBase
 
-# Register your models here.
+from . import models as products_models
+
+
+class MultiDBModelAdmin(admin.ModelAdmin):
```

Comment:
It'd be worth it to declare this class in another file along with the required subclasses so that you can simply import it here and in `products/offers` instead of fully redefining it; You'd only need to override the `using` value as needed, e.g.:

products/admin_base.py
```
class MultiDBModelAdmin(admin.ModelAdmin):
    using = None
    [...]

class OffersDBModelAdmin(admin.ModelAdmin):
    using = 'offers'
```

offers/admin.py:
```
from products.admin_base import OffersDBModelAdmin

admin.site.register(Offer, OffersDBModelAdmin)
```

Reply:
Good point. I have created an abstract class in this commit: https://github.com/lkulbacki/datality-admin-backend/commit/f154fe0733d5cf38cdf91a4a5ee131f4f82f2ebd